spec/models/spree/slide_spec.rb

```diff
@@ -0,0 +1,120 @@
+require_relative '../../spec_helper.rb'
+
+describe Spree::Slide do
+  describe '.published' do
+    let!(:slide_1) { create(:image_slide) }
```

Comment:
You create the same slides multiple times, perhaps we can extract them to the higher context so we will not initialize same values in different sections?

Reply:
fixed in https://github.com/spree-contrib/spree_slider/pull/50/files